app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,52 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+  version :thumb do
+  process :resize_to_fill => [50, 50]
+end
```

Comment:
I don't think this `end` belongs here, it looks like a typo as there is more code defined below.